ConsumerEdition/DragonBoard-410c/Guides/bootable-sd-card.md

```diff
@@ -81,3 +82,40 @@ Where:
 * you can check the mapping between partition IDs and partition names using `gdisk -l` command
 
 If you insert the SD card into the DragonBoard410c and set the switch S6-2 (on the back) to ON, you should now be able to boot from SD card.
+
+# 4) Booting Debian image
+
+Download boot and root images from:
+  - **Release:** http://builds.96boards.org/releases/dragonboard410c/linaro/debian/latest/
+  - **Snapshot:** http://builds.96boards.org/releases/dragonboard410c/linaro/debian/latest/
```

Comment:
wrong link here

Reply:
that seems correct, I'll mention the specific file names they have to download

Reply:
oh ok, got it.